source/_integrations/ozw.markdown

```diff
@@ -65,3 +65,24 @@ this operation.
 | Service Data Attribute | Required | Description                                        |
 | ---------------------- | -------- | -------------------------------------------------- |
 | `instance_id`          | no       | The OZW Instance/Controller to use, defaults to 1. |
+
+### Service `ozw.set_usercode`
+
+This service will set the usercode to X at code_slot Y.
+Valid usercodes are at least 4 digits.
+
+| Service Data Attribute | Required | Description                             |
+| ---------------------- | -------- | --------------------------------------- |
+| `entity_id`            | yes      | Lock entity to set the usercode.        |
```

Comment:
This service uses `async_register_entity_service` to register, thus the `entity_id` is optional as an area can be targeted instead.

Reply:
I'll submit a PR with the requested changes.